modules/preview-react/side-panel/stories/examples/Basic.tsx

```diff
@@ -36,7 +31,15 @@ export const Basic = () => {
               <Flex marginInlineEnd="s">
                 <AccentIcon icon={rocketIcon} />
               </Flex>
-              <StyledHeader {...labelProps}>Tasks Panel</StyledHeader>
+              <TypeBodyLevel
+                as="h3"
+                size="large"
+                color="licorice500"
+                fontWeight="bold"
```

Comment:
if you use the `Heading` component is the font weight automatically `bold`

Reply:
yes, but it's only one prop that matches `Heading`.